easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.13.1-foss-2019a-Python-3.7.2.eb

```diff
@@ -22,6 +22,10 @@ dependencies = [
     ('h5py', '2.9.0'),
 ]
 
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
```

Comment:
@Flamefire Please also clean up the `source_urls` in `exts_list`, at least if they point to PyPI...

Reply:
Done